pkgs/development/python-modules/verilogae/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, fetchurl, buildPythonPackage, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "verilogae";
+  version = "1.0.0";
+  src = fetchurl {
+    url =
+      "https://files.pythonhosted.org/packages/aa/a6/9daea00745844faba44c2917c66838adfc315269f38518ecca49e6eb5fb5/verilogae-1.0.0-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl";
+    sha256 = "QsEVq/4c44xCkovOhXOj8Zh6rp4Ipq+NGjbTW25Fd6E=";
+  };
+  format = "wheel";
+
+  meta = {
+    description =
+      "VerilogAE is a Verilog-A tool useful for compact model parameter extraction.";
+    license = lib.licenses.mit;
```

```suggestion
    license = lib.licenses.gpl3Only;
```
Please respect the license.
https://github.com/pascalkuthe/OpenVAF/blob/a9697ae7780518f021f9f64e819b3a57033bd39f/setup.py#L22